Based off the BWA patch in https://github.com/lh3/bwa/pull/267
Patch by Simon Branford (University of Birmingham)
diff --git a/SeqLib/bwa/rle.h b/SeqLib/bwa/rle.h
index 0d594846..4f8946dc 100644
--- a/SeqLib/bwa/rle.h
+++ b/SeqLib/bwa/rle.h
@@ -30,7 +30,7 @@ extern "C" {
  *** 43+3 codec ***
  ******************/
 
-const uint8_t rle_auxtab[8];
+extern const uint8_t rle_auxtab[8];
 
 #define RLE_MIN_SPACE 18
 #define rle_nptr(block) ((uint16_t*)(block))

diff --git a/SeqLib/fermi-lite/rle.h b/SeqLib/fermi-lite/rle.h
index 0d594846..4f8946dc 100644
--- a/SeqLib/fermi-lite/rle.h
+++ b/SeqLib/fermi-lite/rle.h
@@ -30,7 +30,7 @@ extern "C" {
  *** 43+3 codec ***
  ******************/
 
-const uint8_t rle_auxtab[8];
+extern const uint8_t rle_auxtab[8];
 
 #define RLE_MIN_SPACE 18
 #define rle_nptr(block) ((uint16_t*)(block))

